plugins/basicmessage_storage/README.md

```diff
@@ -0,0 +1,29 @@
+### developer notes
+
+- install python 3.6.13
+- install poetry version 1.1.11
+- the base docker image is python 3.6.13, so best to develop using that version
+- there are issues using poetry lock files generated by a newer version, then run using an older version. the dockerfile
+  uses 1.1.11, so stick with that
+
+### build and run
+
+```
+cd docker
+docker build -f ./Dockerfile --tag basicmessage_storage ..
+docker run -it -p 3000:3000 -p 3001:3001 --rm basicmessage_storage
+```
+
+### notes
+
+there are 2 implementations:
+
+- v0_1: it implements is own version of the `connections/{id}/send-message` endpoint which will persist the sent
+  message. in order to "overwrite" the endpoint, it requires blocking
+  the `aries_cloudagent.aries_cloudagent.protocols.basicmessage` plugin and then registering its messages types,
+- v0_2: uses a middleware wrapper around the existing basicmessage `connections/{id}/send-message` api, and will persist
+  the sent message.
+
+in this case, the `v0_2` pattern is probably best, as we are not changing the signature
+of `connections/{id}/send-message`.
+
```

Comment:
Is this still true? i only see one set of code within this plugin?

Reply:
great catch, that is not true.
will update readme to reflect the code.